src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.figures;
+
+public abstract class Figure implements Drawable {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the class declaration.